packages/portal/src/components/search/SearchInterface.vue

```diff
@@ -399,16 +401,18 @@
       //       `onClickItem`
       // TODO: reduce cognitive complexity
       deriveApiParams() {
-        const params = ['boost', 'qf', 'query', 'reusability', 'sort'].reduce((memo, field) => {
-          if (this[field] && (!Array.isArray(this[field]) || this[field].length > 0)) {
-            memo[field] = this[field];
-          }
-          return memo;
-        }, {});
-
-        params.page = this.page;
-        params.profile = 'minimal';
-        params.rows = this.perPage;
+        const localParams = omitBy({
+          boost: this.boost,
+          qf: this.qf,
+          query: this.query,
+          reusability: this.reusability,
+          sort: this.sort,
+          page: this.page,
+          profile: 'minimal',
+          rows: this.perPage
+        }, isUndefined);
+
+        const params = merge(this.overrideParams, localParams);
```

Comment:
Rename `overrideParams` as it's not overriding, but rather setting defaults.

Reply:
Note to self: or `initialParams`